### ISSUE TITLE
NodeBB crashing when accessing settings on user profile

### ISSUE THREAD

**Issue by Radiergummi** [Schamper/nodebb-plugin-shoutbox#77]
Log shows this:
```

14/5 11:22 [1271] - error: TypeError: undefined is not a function
    at /var/www/node/nodebb/node_modules/nodebb-plugin-shoutbox/library.js:105:12
    at /var/www/node/nodebb/node_modules/templates.js/lib/templates.js:159:12
    at fs.js:336:14
    at FSReqWrap.oncomplete (fs.js:99:15)
TypeError: undefined is not a function
    at /var/www/node/nodebb/node_modules/nodebb-plugin-shoutbox/library.js:105:12
    at /var/www/node/nodebb/node_modules/templates.js/lib/templates.js:159:12
    at fs.js:336:14
    at FSReqWrap.oncomplete (fs.js:99:15)
14/5 11:22 [1271] - info: [app] Shutdown (SIGTERM/SIGINT) Initialised.
14/5 11:22 [1271] - info: [app] Database connection closed.
14/5 11:22 [1271] - info: [app] Web server closed to connections.
14/5 11:22 [1271] - info: [app] Shutdown complete.
[cluster] Child Process (1271) has exited (code: 1, signal: null)
[cluster] Spinning up another process...

`
``

**Comment by Schamper** [comment]
This is already fixed in the rewrite. (Coming soon)

**Comment by bberry6** [comment]
I'm still receiving this.... any update?

**Comment by Schamper** [comment]
No update yet.

**Comment by bberry6** [comment]
Oh nevermind, i just noticed the "development" branch...

I'm just getting into nodebb and plugin dev for it.  If I wanted to help in tackling some of these bugs/feature requests, I assume I should start by sending PRs for the development branch.  What would I need to do to get that branch working locally for me (as I saw that in your commit message, you say something on the lines of "dont use this")?

**Comment by Schamper** [comment]
Yeah the development branch is very much a work in progress. Although you might've noticed that the progress part has kind of stalled lately.

The "dont use this" message means that it's not ready for production use yet. There's still a bunch of fixing, tweaking and cleanup necessary to get it up to par with the master branch. (In case you haven't noticed, the development branch is a pretty large refactor)

Feel free to play around with it and see if you can fix some stuff or improve my existing code. I'll get back to working on it myself eventually (Soon™, amirite?).

**Comment by bberry6** [comment]
haha sounds good.

**Comment by Schamper** [comment]
Fixed with v0.3.0.
